Project.toml

```diff
@@ -26,7 +26,7 @@ DataFrames = "1"
 DataStructures = "0.18"
 InfrastructureSystems = "1"
 Plots = "1"
-PowerAnalytics = "0.3"
+PowerAnalytics = "^0.3.3"
```

Comment:
Bump to 0.4 once it is registered check https://github.com/JuliaRegistries/General/pull/89039